contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -2199,8 +2199,16 @@ LEFT JOIN sys.babelfish_server_options AS s on f.srvname = s.servername
 WHERE w.fdwname = 'tds_fdw';
 GRANT SELECT ON sys.servers TO PUBLIC;
 
+CREATE OR REPLACE FUNCTION sys.openquery_internal(
+IN linked_server text,
+IN query text)
+RETURNS SETOF RECORD
+AS 'babelfishpg_tsql', 'openquery_internal'
+LANGUAGE C VOLATILE;
+
 CALL sys.babelfish_drop_deprecated_object('procedure', 'sys', 'babelfish_sp_rename_internal_deprecated_in_3_2_0');
 CALL sys.babelfish_drop_deprecated_object('procedure', 'sys', 'sp_rename_deprecated_in_3_2_0');
+CALL sys.babelfish_drop_deprecated_object('function', 'sys', 'openquery');
```

Comment:
if this fails due to use side dependency, is ok to have dangling open query function? is there a way use call openquery directly without it being re-written to openquery_internal?

Reply:
I think it is fine because both sys.openquery() and sys.openquery_internal() do the exact same thing. There is no way to call T-SQL OPENQUERY() directly without it being re-written to openquery_internal.

Reply:
I think we need an additional test to ensure the deprecated `sys.openquery` works as expected even after the upgrade, i.e., create a dependent object of `sys.openquery` in `15.2` and call it after mVU to the latest.